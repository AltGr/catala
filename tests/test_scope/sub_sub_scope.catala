/*
declaration scope A:
  context x content integer
  context u content boolean

declaration scope B:
  context a scope A
  context y content integer

declaration scope C:
  context a scope A
  context b scope B
  context z content integer

scope A:
 definition x equals 0
 definition u equals true

scope B:
  definition a.x under condition a.u consequence equals 1
  definition y under condition a.x = 1 consequence equals 1
  definition y under condition a.x + 1 = 2 consequence equals 1

scope C:
  definition a.x equals 2
  definition b.y equals 3
  definition z equals 2
*/
