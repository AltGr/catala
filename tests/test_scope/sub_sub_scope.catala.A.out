[RESULT] u -> true
[RESULT] x -> 0
